allen-cell-animated/cell-feature-data
# Upgrade to pyproject.toml

Problem
=======
What is the problem this work solves, including
closes #104 

Estimate review time: 5mins (tiny)

Solution
========
What I/we did to solve this problem
- updated all `questionary` `ask` functions that were previously missed to `unsafe_ask`  
- removed `setup.py`
- added `pyproject.toml`
- tested with new environment 

Todo next: 
- add more necessary project metadata
- decide on a license 

## Type of change
Please delete options that are not relevant.

* New feature (non-breaking change which adds functionality)


Steps to Verify:
----------------
1. Follow the [instruction](https://github.com/allen-cell-animated/cell-feature-data/tree/feature/python-script?tab=readme-ov-file#option-1-use-the-python_dataset_creator-package) in README
2. This change shouldn't affect any existing functionalities. Running `create-dataset` should work the same
